### PR TITLE
Nectar CRM [APP] #3072

### PR DESCRIPTION
**Pull request by lcaresia**

### CODE REVIEW

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible

File: components/nectar_crm/actions/create-contact/create-contact.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      data: {
+        ativo: this.active,
+        nome: this.name,
+        cnpj: this.cnpj,
+        origem: this.origin,
+        categoria: this.category,
+        constante: this.type,
+        observacao: this.observation,
+        emails: [
+          this.email,
```
Comment (jcortes) [inline components/nectar_crm/actions/create-contact/create-contact.ts:82]: Sincie `email` is optional what happens if the user doesn't set this field? the system is able to handle the `emails: [undefined]` here?

**Review by jcortes — COMMENTED** [review]
Hi @lcaresia I've just added one question, the rest looks good to me!

**Comment by vunguyenhung** [comment]
Hello @lcaresia, I'm just checking in, do you have bandwidth to resolve this as there's a request change from jcortes 3 days ago

**Comment by lcaresia** [comment]
@vunguyenhung My trial expired, I'm in contact with NectarCRM commercial team to resolve this!

**Comment by vunguyenhung** [comment]
Thank you for the updates @lcaresia! Hopefully NectarCRM team can unblock us to keep this ticket resolved

File: components/nectar_crm/actions/create-contact/create-contact.ts (hunk 1 above)
Comment (lcaresia) [reply components/nectar_crm/actions/create-contact/create-contact.ts:81]: Just is needed the `name` prop, fixed!

**Review by jcortes — COMMENTED** [review]
HI @lcaresia LGTM! Nothing to add, nice job! Ready for QA!

**Comment by DilanAthukorala** [comment]
Hi @lcaresia Verification completed

please correct small typo here before releasing. should be "Create" instead of created.
image

**Review by jcortes — APPROVED** [review]
Approved!

**Pull request merged**